lib/Sema/TypeCheckAttr.cpp

```diff
@@ -842,6 +843,7 @@ class AttributeChecker : public AttributeVisitor<AttributeChecker> {
     IGNORED_ATTR(Transparent)
     IGNORED_ATTR(WarnUnqualifiedAccess)
     IGNORED_ATTR(WeakLinked)
+    IGNORED_ATTR(UsableFromInlineImport)
```

Comment:
Nitpick: This particular list is alphabetized; please keep it that way.

Reply:
Done